src/Aspire.Dashboard/Components/Pages/Metrics.razor

```diff
@@ -91,7 +103,17 @@
                                     }
                                     else
                                     {
-                                        <p>@Loc[nameof(Dashboard.Resources.Metrics.MetricsSelectInstrument)]</p>
+                                        @if (ViewportInformation.IsDesktop)
+                                        {
+                                            <p>@Loc[nameof(Dashboard.Resources.Metrics.MetricsSelectInstrument)]</p>
+                                        }
+                                        else
+                                        {
+                                            // class needed to prevent color from being removed from reboot style
+                                            <a class @onclick="@(async () => await _contentLayout!.OpenMobileToolbarAsync())">
```

Comment:
What's going on here with class? An attribute without a value is wierd

Reply:
HTML supports empty attributes: https://html.spec.whatwg.org/multipage/syntax.html#attributes-2

Reply:
Yeah but Razor doesn't like it. There are IDE warnings that the attribute is missing a value. I'd prefer `class=""` to remove the warning.

I'm not sure why `class` (or `class=""`) would have an impact. Can you have CSS styles that are applied only when there is no class?

Reply:
> Can you have CSS styles that are applied only when there is no class?

yes actually, you can - and the default reboot styling removes the link color unless the link tag has a class attribute

Reply:
changed to class=""